packages/browser-wallet/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@concordium/browser-wallet",
     "private": true,
-    "version": "1.3.2",
+    "version": "1.4.0",
```

Comment:
Would this not be a patch release?

Reply:
Yes, but what you don't see is that I have already merged in some new functionality in a different PR. I just bumped the version here to avoid another PR.